analysis/utility_analysis_engine.py

```diff
@@ -105,17 +105,17 @@ def _create_compound_combiner(
                 internal_combiners.append(
                     utility_analysis_combiners.PartitionSelectionCombiner(
                         combiners.CombinerParams(budget, params)))
-            if pipeline_dp.Metrics.COUNT in aggregate_params.metrics:
+            if pipeline_dp.Metrics.SUM in aggregate_params.metrics:
```

Comment:
Just wondering, why the order is changed?

Reply:
The order needs to be the same in both places (utility_analysis_engine.py and utility_analysis.py), so the order *is* important.

Given the order is important, I thought this order would make a bit of sense (going from the most complicated aggregation to least complicated). Similarly, the inverse would also make a bit of sense.

Reply:
Ok, SG, thanks, for explanations!